client/src/test/java/tds/session/SessionTest.java

```diff
@@ -19,4 +22,64 @@ public void shouldBeInstantiated() {
         assertThat(session.getType()).isEqualTo(0);
         assertThat(session.getStatus()).isEqualTo("closed");
     }
+
+    @Test
+    public void shouldBeOpen() {
```

Comment:
I am more used to the shouldDoThisAndThat naming convention.  The it naming reminds me of how Jasmine tests are written when doing Angular unit tests so I think I associate it with JavaScript testing in my mind.  If no one has a strong opinion then let's go with should instead of itShould and save 2 valuable characters and keep the file sizes down ;)